ssh/ssh.go

```diff
@@ -164,6 +168,9 @@ var _ = SshDescribe("SSH", func() {
 		})
 
 		It("records successful ssh attempts", func() {
+
+			Skip("Eirini doesn't log ssh access")
```

Comment:
This could be added to the eirini-ssh extension if we think it's useful. For now skipping the test is ok since we know exactly what the problem is.